docs/hocs/query.md

```diff
@@ -0,0 +1,63 @@
+# query
+
+An higher-order component to [Relay QueryRenderer](https://facebook.github.io/relay/docs/en/query-renderer.html).
+
+## Usage
+
+```javascript
+import { graphql } from 'react-relay';
+import { query } from 'react-releasy';
+
+// ... more code here
```

Comment:
Detail here the code to be clearer

Reply:
this won't be necessary for now, we have a lot of examples at the root `README.md` and there's nothing different from a simple React component